docs/integrations/tax.mdx

```diff
@@ -162,6 +162,12 @@ Avoid synchronous calls to external systems when preparing your tax estimate res
 
 Using [Tax Properties](/docs/store-operations/tax/tax-properties) to enhance the initial Tax Provider API request can help you avoid looking up additional tax estimate inputs on external systems.
 
+#### Infinite loops
```

Comment:
🍻 Unsure if there is a better terminology here.

Reply:
I think it is fine, given that the audience should be a developer.

Reply:
I wonder, and this may be silly, do feel free to ignore. But what about `Avoiding infinite loops`

Reply:
Oh thanks, I must have pressed "Resolve conversation" at the exact same time as your suggestion. Love it. Cheers.